### PR TITLE
font-iosevka-ttf: Update to 27.2.0

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Add characters
  * TELEPHONE RECORDER (`U+2315`) (#2020).
  * COUNTERSINK (`U+2335`) (#2020).
  * BROKEN CIRCLE WITH NORTHWEST ARROW (`U+238B`) (#2020).
  * TRIPLE PLUS (`U+29FB`) (#2020).
- Correction of letter assignments for stylistic styles:
  * Fix `cv10` and `cv30` for `ss14`.
  * Fix `cv12`, `cv36`, and `cv82` for `ss16`.
  * Fix `cv22`, `cv31`, `cv47`, and `cv91` for `ss07`.
  * Fix `cv26` for `ss07`, `ss14`, and `ss15`.
  * Fix `cv28`, `cv43`, and `cv70` for `ss01`, `ss04`, `ss05`, `ss06`, `ss07`, `ss13`, and `ss16`.
  * Fix `cv55` for `ss03`, `ss09`, `cv12`, and `ss14`.
  * Fix `cv53` and `cv84` for `ss10`.
  * Fix `cv58` for `ss06` and `ss10`.
  * Fix `cv59` for `ss09` and `ss18`.
  * Fix `cv61` for `ss08` and `ss20`.
  * Fix `cv62` for `ss01`.
  * Fix `cv63` for `ss02`, `ss05`, and `ss12`.
  * Fix `cv68` for `ss03`.
  * Fix `cv72` for `ss03`, `ss05`, `ss06`, `cv07`, and `ss13`.
  * Fix `cv77` for `ss03`, `ss04`, `ss06`, `ss07`, `ss08`, `ss12`, `ss13`, and `ss18`.
  * Fix `cv79` and `cv80` for `ss01`, `ss02`, `ss04`, `ss05`, `ss06`, `ss07`, `ss09`, `ss12`, `ss13`, `ss14`, `ss16`, `ss18`, and Aile.
  * Fix `cv88` for `ss18`.
  * Fix `vsAA` and `vsAM` for `ss06`.
  * Fix `vsAG` for `ss03` and Etoile.
  * [changelog](https://raw.githubusercontent.com/be5invis/Iosevka/v27.2.0/CHANGELOG.md)
- Including `metainfo.xml` (Part of getsolus#449)

**Test Plan**

- Render text with the font
- Build the package
- Run `appstream-builder --packages-dir=. --include-failed -v`
- Check that the package doesn't show up in the `example-failed.xml.gz`

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/font-iosevka-ttf/files/iosevka.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>iosevka</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Iosevka</name>
+  <homepage>https://typeof.net/Iosevka/</homepage>
```
Comment (joebonrichie) [inline packages/f/font-iosevka-ttf/files/iosevka.metainfo.xml:7]: Should be:
```suggestion
  <url type="homepage">https://typeof.net/Iosevka/</url>
```

I know you copied it off one of my faulty commits, sorry about that! Please amend the commit and force push.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url

**Review by joebonrichie — CHANGES_REQUESTED** [review]
One small thing

File: packages/f/font-iosevka-ttf/files/iosevka.metainfo.xml (hunk 1 above)
Comment (malfisya) [reply packages/f/font-iosevka-ttf/files/iosevka.metainfo.xml]: Done

**Review by ReillyBrogan — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**